website/src/writing-guide/styles/Replace/Offensive.yml

```diff
@@ -0,0 +1,27 @@
+extends: substitution
+message: "'%s' is offensive, and has been censored."
+ignorecase: true
+level: error
+action:
+  name: replace
+swap:
+  fag: [CENSORED]
```

Comment:
Had to change these to a string `[CENSORED]->CENSORED` for it to run locally. Probably due to versions gap.

Reply:
That makes sense. Thank you for updating it.